### PR TITLE
Add inline convert.

### PR DESCRIPTION
**Pull request by tylawin**
Adds conversion for tabs to spaces or spaces to tabs inside a line.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -296,6 +299,59 @@ inline string& remove_trailing_whitespaces(string& line)
...
+		{
+			exploded.push_back(str);
+			return exploded;
+		}
+		else
+		{
+			exploded.push_back(str.substr(0, pos));
+			str = str.substr(pos);
+		}
+	}
```
Comment (vaehx) [inline main.cpp:345]: Why not use state bit to shorten this code and avoid duplicate code:

``` cpp
size_t pos;
bool findNextChar = (str[0] == ' ' || str[0] == '\t');
while (!str.empty())
{
    if (findNextChar)
        pos = str.find_first_not_of(" \t");
    else
        pos = str.find_first_of(" \t");

    if (pos == string::npos)
    {
        exploded.push_back(str);
        return exploded;
    }
    else
    {
        exploded.push_back(str.substr(0, pos));
        str = str.substr(pos);
    }

    findNextChar = !findNextChar;
}
```

**Comment by vaehx** [comment]
IMHO inline whitespace conversion should not be enabled by default.
Or maybe add an option to disable it

**Comment by tylawin** [comment]
Since it isn't smart enough to avoid changes inside quotes it is probably best to have the default be disabled.

File: main.cpp (hunk 1 above)
Comment (tylawin) [inline main.cpp:345]: Sure, I'll try to give findNextChar a slightly better name though.

File: README.md [hunk 2]
```diff
@@ -3,8 +3,9 @@ Command-line sourcefile formatting tool.
 This tool adds a new empty line at end of file by default, if not there yet.
 Also removes trailing whitespaces by default.
 
-* Replace space indent with tab indent: `sff yourfile.txt`
+* Replace space indent with tab indent and tabs in line to spaces: `sff yourfile.txt`
 * Replace tab indent with space indent: `sff -s yourfile.txt`
+* Replace spaces inside line with tabs: `sff -t yourfile.txt`
```
Comment (vaehx) [inline README.md]: should be replaced by the `-s2t` and `-t2s` options

**Comment by vaehx** [comment]
lgtm

**Pull request merged**